phpBB/config/installer/routing/installer.yml

```diff
@@ -36,31 +36,6 @@ phpbb_installer_update_conflict_download:
     defaults:
         _controller: phpbb.installer.controller.file_downloader:conflict_archive
 
-phpbb_convert_intro:
```

Comment:
Won't removing the whole framework break all other existing converters as well? Is that something we really want?

Reply:
They can convert to 3.3.x then upgrade to 4.0.x?
Btw the old convertors are really very old.

Reply:
Well, it can be added later again, anyway there is no other converters.
The code can be added later again if there is need

Reply:
And what about the custom user made converters from board software "abc" to phpBB? Are they not depending on this framework?

Reply:
> And what about the custom user made converters from board software "abc" to phpBB? Are they not depending on this framework?

Yes, they are, and that is where my comment is from. By removing all of this we are removing support for all existing converters, and while they might be old, this might not be something we really want.

Reply:
> 
> 
> They can convert to 3.3.x then upgrade to 4.0.x?
> Btw the old convertors are really very old.

Reply:
The converters as a new and more modern framework it was a part of some GSOC in the past but the result was equal to null. No need IMHO to have all of this code for nothing.

Reply:
I can remove only the phpbb2 converter if you want, but remove everything is less code to maintain